Instalación y configuración

Paso 1: Verificar si Tkinter está instalado.

Para comprobar si Tkinter está disponible en el entorno de desarrollo Python, se puede ejecutar el siguiente código:

In [3]:
import tkinter as tk
print(tk.TkVersion)

8.6


Si Tkinter está instalado, mostrará la versión (generalmente 8.6 o superior).

Paso 2: Instalación (si es necesario)

Si Tkinter no está instalado, se puede instalar utilizando el siguiente comando en la consola o terminal:

*bash*

pip install tk


En sistemas Linux, también puede ser necesario instalar el paquete:

*bash*

sudo apt-get install python3-tk
